SuperBuild.cmake

```diff
@@ -32,6 +46,7 @@ ExternalProject_Add(Autoscoper
     -DTIFF_LIBRARY:FILEPATH=${TIFF_LIBRARY}
     -DTIFF_INCLUDE_DIR:PATH=${TIFF_INCLUDE_DIR}
```

Comment:
Remove 

```
    -DGLEW_DIR:PATH=${GLEW_DIR}
    -DTIFF_LIBRARY:FILEPATH=${TIFF_LIBRARY}
    -DTIFF_INCLUDE_DIR:PATH=${TIFF_INCLUDE_DIR}
```

These can be removed thanks to the use of `mark_as_superbuild` in the associated `External_*.cmake` files